Mathlib/Analysis/NormedSpace/Star/Spectrum.lean

```diff
@@ -69,6 +69,10 @@ theorem IsSelfAdjoint.spectralRadius_eq_nnnorm {a : A} (ha : IsSelfAdjoint a) :
   simp
 #align is_self_adjoint.spectral_radius_eq_nnnorm IsSelfAdjoint.spectralRadius_eq_nnnorm
 
+lemma IsSelfAdjoint.toReal_spectralRadius_complex_eq_norm {a : A} (ha : IsSelfAdjoint a) :
```

Comment:
Please add a note in a docstring pointing the `ℝ` version, so that people know it exists.